concourse/model/traits/component_descriptor.py

```diff
@@ -47,10 +47,13 @@ class ValidationPolicy(EnumWithDocumentation):
     )
 
 
+DEFAULT_COMPONENT_DESCRIPTOR_STEP_NAME = 'component_descriptor'
```

Comment:
nit: it is actually _the_ component-descriptor-step-name (we cannot change this anymore easily)

Reply:
We cannot (or rather should not), but it is a default that can be overridden by the User (see l.56 below)